src/yb/server/server_base_options.cc

```diff
@@ -102,6 +102,22 @@ ServerBaseOptions::ServerBaseOptions(int default_port)
     LOG_IF(DFATAL, !status.ok()) << "Bad public IPs " << FLAGS_server_broadcast_addresses
                                  << ": " << status;
   }
+
+  LOG_IF(INFO, CheckSpaces(FLAGS_placement_uuid)) << 
+          "FLAGS_placement_uuid has leading or trailing spaces,should trim it";
+  placement_uuid = yb::util::TrimStr(FLAGS_placement_uuid);
+
+  LOG_IF(INFO, CheckSpaces(FLAGS_placement_cloud)) << 
+          "FLAGS_placement_cloud has leading or trailing spaces,should trim it";
+  placement_cloud_ = yb::util::TrimStr(FLAGS_placement_cloud);
+
+  LOG_IF(INFO, CheckSpaces(FLAGS_placement_region)) << 
+          "FLAGS_placement_region has leading or trailing spaces,should trim it";
+  placement_region_ = yb::util::TrimStr(FLAGS_placement_region);
+
+  LOG_IF(INFO, CheckSpaces(FLAGS_placement_zone)) << 
+          "FLAGS_placement_zone has leading or trailing spaces,should trim it";
+  placement_zone_ = yb::util::TrimStr(FLAGS_placement_zone);
```

Comment:
I don't think we even need to check if theres spaces in the string first, lets just call TrimStr on the placement flags.

Reply:
Yes ,  just call the TrimStr is okay  .   I was intent to log the msgs where these placements contain spaces.

Reply:
If no need to log the msgs , i would remove the check codes